pkg/artifactory/resource_artifactory_remote_repository_test.go

```diff
@@ -37,6 +37,60 @@ func TestAccRemoteRepository_basic(t *testing.T) {
 	})
 }
 
+const remoteRepoNuget = `
+resource "artifactory_remote_repository" "terraform-remote-test-repo-nuget" {
+	key = "terraform-remote-test-repo-nuget"
+    package_type                          = "nuget"
+		feed_context_path = "/api/v2"
+		v3_feed_url       = "https://api.nuget.org/v3/index.json"
+}`
+
+func TestAccRemoteRepository_nuget(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		CheckDestroy: resourceRemoteRepositoryCheckDestroy("artifactory_remote_repository.terraform-remote-test-repo-nuget"),
+		Providers:    testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: remoteRepoNuget,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("artifactory_remote_repository.terraform-remote-test-repo-nuget", "key", "terraform-remote-test-repo-nuget"),
+					resource.TestCheckResourceAttr("artifactory_remote_repository.terraform-remote-test-repo-nuget", "v3_feed_url", "https://api.nuget.org/v3/index.json"),
+					resource.TestCheckResourceAttr("artifactory_remote_repository.terraform-remote-test-repo-nuget", "feed_context_path", "/api/v2"),
+				),
+			},
+		},
+	})
+}
+
+const remoteRepoDeprecatedNuget = `
+resource "artifactory_remote_repository" "terraform-remote-test-repo-nuget" {
+	key = "terraform-remote-test-repo-nuget"
+		package_type                          = "nuget"
+		nuget = {
+			feed_context_path = "/api/v2"
+			v3_feed_url       = "https://api.nuget.org/v3/index.json"
+		}
+}`
+
+func TestAccRemoteRepository_nuget(t *testing.T) {
```

Comment:
This function has a duplicate name to line 48 which is breaking the build. If we rename this, it is ready to merge straight away.